pkg/errors/errors.go

```diff
@@ -38,6 +38,9 @@ var (
 	alreadyExistsErrorCodes = sets.New[string](
 		"EntityAlreadyExists",
 	)
+	accessDeniedErrorCodes = sets.New[string](
+		"AccessDeniedException",
```

Comment:
Are we sure this is the only error code that we'd get? I imagine this might change. Can we do it based off the error code number?